### DISCUSSION TITLE
Input name attribute not being used

### DISCUSSION THREAD

**Discussion by gomarquestsr** [tamagui/tamagui#3019] [General]
### Current Behavior

Input component is not rendering the `name` attribute.

I'm running this:
```
import { Input } from 'tamagui';

const Test = () => (
  <Input name="input name" />;
);
export default Test;
```

and it is rendering this html:
```<input autocapitalize="sentences" autocomplete="on" autocorrect="on" dir="auto" rows="1" spellcheck="true" virtualkeyboardpolicy="auto" data-disable-theme="true" tabindex="0" class="...">```

### Expected Behavior

It is expected that the `name` prop gets passed to Input element:
```<input autocapitalize="sentences" autocomplete="on" autocorrect="on" dir="auto" rows="1" spellcheck="true" virtualkeyboardpolicy="auto" data-disable-theme="true" tabindex="0" class="..." name="input name">```

### Tamagui Version

```markdown
v1.115.4
```

### Platform (Web, iOS, Android)

```markdown
Web, iOS, Android
```

### Reproduction

```markdown
import { Input } from 'tamagui';

const Test = () => (
  <Input name="input name" />;
);
export default Test;
```

**Comment by natew** [comment]
We simply wrap react-native-web, which has this limitation.

You can try our re-write `@tamagui/input-next` for now. Moving to discussion.
